js/modal.js

```diff
@@ -76,6 +79,7 @@
           var oldWidth = $body.innerWidth();
           $body.css('overflow', 'hidden');
           $body.width(oldWidth);
+          $('html').css('overflow', 'hidden');
```

Comment:
I'll clean that up.  I could instead, if you'd rather, make these changes by adding a custom class to `<html>` instead of injecting inline css.  I believe that would be a less intrusive way to add the styling.